doc/guidelines/frontend-ui-usability-guide.md

```diff
@@ -0,0 +1,166 @@
+- **Double submission** should be **prevented**, e.g. by disabling the submission button until the server has responded.
+
+### Consistency
+
+Things with the **same meaning** should **look and behave** the **same** throughout the whole application.
+Users should never be uncertain if different words mean the same thing or different actions trigger the same process.
+
+Examples:
+
+- Call-to-action buttons should look the same everywhere.
+- The layout of forms and the styling of their input fields should be the same in every form.
+- Validation errors in forms should be rendered the same way in any form.
+- Toast errors should always be shown in the same location.
+- Headings of different pages should be positioned in the same location and have the same styling.
+- The same application-specific terms should be used everywhere to communicate to the user.
+
+### Icons
+
+- In their **passive**, i.e. currently not active/enabled, state they should appear in **neutral** colours/colourless.
+- When **active**, i.e. currently activated/enabled, they should get a small **highlighting** through a visually stronger and more colourful appearance by using the design's primary/secondary colours.
+- When **disabled**, i.e. currently not clickable, they should be **greyed-out**, either via decreasing its opacity or choosing light, faint colours.  
+  The mouse **cursor** should be styled with the **_not-allowed_** css property.
+- A **tooltip** on hovering should be displayed for every icon in PermaplanT's toolbox.
+  The tooltip should contain the icon's **label** (as concise as possible) and, if existing, the assigned **shortcut**.
+
+### Highlighting
+
+Highlighting techniques:
+
+- Font weight:
+
+  - **Bold** text adds tolerable noise to the design and enables distinct highlighting in both short and long texts.
+    In general, it is the preferred highlighting technique to use.
+  - _Italic_ text adds minimal noise to the design, but lacks in recognizability.
+  - <u>Underlining</u> adds most noise and compromises a text's legibility.
+    It should hardly ever be used.
+
+- Colours: primary and secondary colours of the design's colour palette
+
+- Images: users generally remember images better than words
+
+- Avoid different fonts for highlighting purposes.
+
+### Wording
+
+Messages shown to the user should strive to fulfill following criteria:
+
+- **concise**: the more text the less likely it will be read by the user
+- **clear**: straight to the message's essence
+- **understandable** for technical laymen: no error codes or technical terms
+- **no exclamation** marks: interpreted as commanding
+- **no uppercase** words: comes across as shouting
+- **detailed** information should be hidden behind a **read more** link or a **collapsed** section
+- **Headlines**, **sub-headlines** and **labels** should be **capitalized**
+- **Minimize hyphenation** of words
+
+### Error Messages
```

Comment:
Overlap to the other UI document, can you move it here? Do we really want everything in one document?